src/components/document-viewer/doc-fragment.jsx

```diff
@@ -425,9 +425,7 @@ class DocFragment extends Component {
                   {title.indexOf(PENSIEVE_IMPORTED_COPY_FIELD_KEY) === -1 && (
                     <h3>
                       <AnchorLink
-                        text={
-                          this.props.content[this.getTitleKey()] + '-' + title
-                        }
+                        text={this.props.content.getUuid() + '-' + title}
```

Comment:
Does something clever need to be done here (e.g. appending an auto-incrementing number), in case the entry with a given `Uuid` has two headers with the identical `title` ?
(or is that guaranteed to never happen?)

P.S. I obviously know nothing about the framework being used here (and it's outside the scope of this particular PR), but I wonder if `<AnchorLink href=...>` (or `<AnchorLink id=...>`) would have made it more obvious what this is doing than `<AnchorLink text=...>` ? (people might assume that a `text` attribute is what gets displayed to the user?)

Reply:
The data structure used for entries guarantees that you never have a duplicated field header on an entry. Using an attribute like `hash` or `fragment` would be more semantically correct, but is beyond the scope of this PR.

Reply:
> The data structure used for entries guarantees that you never have a duplicated field header on an entry.

Are you sure? How does that account for URLs like:
https://resin-io.github.io/scratchpad/#2aa7ca4c-d424-11e7-a9ca-d3e1e018ddf6-treatment-after-resinos-v1-1-2--this-issue-shouldn-t-happen-as-dnsmasq-is-used-as-the-resolver--

Reply:
@lurch I don't understand what you mean? Does the link go to a duplicated header?

Reply:
Sorry Lucian, my mistake - I thought such a long anchor-link was coming from the ability to use Markdown text inside entries, but looking at the [YAML file](https://github.com/resin-io/scratchpad/blob/master/scratchpad.yaml#L1012) I see that this actually _is_ the name of one of the fields in that entry.
So I guess what you were saying earlier is that even _with_ user-defined fields, pensieve doesn't allow multiple fields in the same entry to have the same name? (apologies if I'm using the wrong terminology)

Reply:
@lurch the entries are esentially defined as JSON objects where the key is the subheader, so as soon as you add a 'header' that already exists, it just overwrites the pre-existing one.

Reply:
How does that manifest itself in the pensieve web-interface? Could a user accidentally overwrite an existing field by creating a new field with the same name? (If yes, should I create a new issue for that?)

Reply:
I've just tested this in the pensieve web-interface myself, and if you add a user-defined field that has the same name as an existing field, then Pensieve doesn't display any error or warning, and any contents in the existing field get silently discarded :-(  I'll add an issue...